token/core/zkatdlog/nogh/v1/crypto/rp/bulletproof.go

```diff
@@ -8,12 +8,12 @@ package rp
 
 import (
 	math "github.com/IBM/mathlib"
+	"github.com/hyperledger-labs/fabric-token-sdk/token/core/common/encoding/asn1"
 	"github.com/hyperledger-labs/fabric-token-sdk/token/core/zkatdlog/nogh/v1/crypto/common"
 	"github.com/pkg/errors"
 )
 
-// RangeProof proves that a committed value < max
-type RangeProof struct {
+type RangeProofBase struct {
```

Comment:
What did we rename this to `RangeProofBase`?

Reply:
Just to simplify the serialization. A bit of syntactic sugar.